drivers/utilities/utilities.go

```diff
@@ -286,3 +294,20 @@ func GenerateRandomCommandToCreateFiles(count int) map[string][]string {
 
 	return randomFileCommands
 }
+
+// syncData will execute 'sync' command on pod post start
+func syncData(namespace string) error {
+	var k8sCore = core.Instance()
+	allPods, err := k8sCore.GetPods(namespace, make(map[string]string))
+
+	for _, pod := range allPods.Items {
+		_, err = k8sCore.RunCommandInPod([]string{"sync"}, pod.Name, pod.Spec.Containers[0].Name, namespace)
```

Comment:
What if there are some pods which do not recognise the `sync` command?

Reply:
In that case instead of failing with error we can print a warning message and proceed with the test, will it work?

Reply:
Yes. We will just print the warn logs. `log.WarnD`